roles/edpm_ovs/molecule/default/verify.yml

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
ill work and updatign this to actully verify ovs installation
this is still just a copy paste form the libvirt role.